### DISCUSSION TITLE
bborgstance confirmation and cost

### DISCUSSION THREAD

**Discussion by rozzingit** [Arx-Game/arxcode#374] [Ideas]
So in light of [this post](https://play.arxgame.org/comms/boards/434/view/1361933), I had an idea. Some commands in the game require a double-confirmation, generally when some currency is being spent (such as org briefing or adding clues to orgs). In order to encourage bborgstances being used in a manner that is thematically appropriate, I wonder if two items might be worth considering:

1) Require a confirmation for posting a bborgstance with a message about the full IC implications of making a bborgstance to begin with. Something like:

> In submitting this org stance, your character will be paying town criers to declare this opinion as a solemn expression of your house or organization's formal, public stance on this proclamation. Submit again to confirm.

2) Make it cost stuff! 25 econ and social was suggested to me, although that's p cheap, tbh. But it might be nice just as a sort of reminder of "there's a social and economic cost to doing this sort of thing." If you did this, naturally just add it to the end of the confirmation message. On the other hand, Proclamations themselves should probably also have a cost, but this particular thread is just about bborgstance, lol.

**Comment by TehomCD** [comment]
This is something staff has talked about too, more or less the same sort of things you're mentioning here. I'm definitely in favor of making it less trivial
